wiki/adr-006-otlp-log-export.md

```
@@ -154,20 +154,38 @@ func (b *appBootstrap) dependencies() *dependencyBundle {
 }
 ```
 
-**6. Deterministic Sampling** (`observability/logs.go`)
+**6. Deterministic Sampling** (`observability/dual_processor.go`)
 ```go
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)